src/SFA.DAS.AssessorService.Application/Handlers/Staff/StaffSearchHandler.cs

```diff
@@ -142,16 +143,16 @@ private List<StaffSearchItems> MatchUpExistingCompletedStandards(List<StaffSearc
             return searchResults;
         }
 
-        private List<StaffSearchItems> PopulateStandards(List<StaffSearchItems> searchResults, IAssessmentOrgsApiClient assessmentOrgsApiClient, ILogger<SearchHandler> logger)
+        private List<StaffSearchItems> PopulateStandards(List<StaffSearchItems> searchResults, IStandardService standardService, ILogger<SearchHandler> logger)
```

Comment:
Could this be made async?

Reply:
yeah, I can do that, and will do.  But this isn't new code, I've just changed the source of data :D